test/unit/kernel/src/KernelWithHostConstexpr.cpp

```diff
@@ -7,9 +7,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-// NVCC needs --expt-relaxed-constexpr
-#if !defined(__NVCC__) || (defined(__NVCC__) && defined(__CUDACC_RELAXED_CONSTEXPR__))
```

Comment:
Not sure if we can remove this. Relaxed constexpr can still be disabled in cmake. This flag allows accessing `__host__ ` constexpr function on the device.

https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#constexpr-functions

Reply:
As discussed below we now enforce this. I will add a compile-time check.